components/ei-analytics/org.wso2.analytics.solutions.ei.analytics/widgets/EIAnalyticsHorizontalBarChart/package.json

```diff
@@ -3,15 +3,17 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "@wso2-dashboards/widget": "1.2.2",
+    "@wso2-dashboards/widget": "1.4.0",
     "css-loader": "^1.0.0",
     "eslint-config-airbnb": "^17.1.0",
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.1",
     "eslint-plugin-react": "^7.11.1",
     "moment": "^2.22.2",
     "react": "^16.4.1",
-    "react-vizgrammar": "^0.7.16",
+    "react-router": "^4.3.1",
```

Comment:
Since we are not using react-router, remove `react-router`, `react-router-dom` dependencies